lib/active_fedora/aggregation/base_extension.rb

```diff
@@ -15,8 +15,17 @@ def aggregated_by
       proxy_class.where(proxyFor_ssim: id).map(&:container)
     end
 
+    def ordered_by
+      ordered_by_ids.map{ |x| ActiveFedora::Base.find(x) }
```

Comment:
Probably, I went for a similar thing as #aggregated_by. Maybe ordered_by_ids.lazy.map?